utoipa-gen/src/component/schema.rs

```diff
@@ -283,22 +284,23 @@ impl NamedStructSchema<'_> {
             .as_ref()
             .map(|features| features.iter().any(|f| matches!(f, Feature::Default(_))))
             .unwrap_or(false);
-        let serde_default = serde::parse_container(self.attributes)
+        let serde_default = container_rules
+            .as_ref()
             .map(|rules| rules.default)
             .unwrap_or(false);
 
         if schema_default || serde_default {
             let field_ident = field.ident.as_ref().unwrap().to_owned();
-            let struct_ident: Ident = syn::parse_str(&self.struct_name).unwrap();
+            let struct_ident = format_ident!("{}", &self.struct_name);
             let default_feature = Feature::Default(
                 crate::component::features::Default::new_default_trait(struct_ident, field_ident),
             );
```

Comment:
Also one thing came to my mind as well. If we make this `default_feature` a closure then we can get rid of the annoying `clone()` below.
```rust
let default_feature = || Feature::Default(...)
```

Reply:
That actually also causes a move ("closure cannot be invoked more than once because it moves the variable `struct_ident` out of its environment"). But to avoid the clone I just realized I can also create an empty vector since the feature would get added right after:
```rust
let features_inner = field_features.get_or_insert(vec![]);
if !features_inner
    .iter()
    .any(|f| matches!(f, Feature::Default(_)))
{
     features_inner.push(default_feature);
}
```
I also cleaned things up a bit since `default_feature` no longer needs to be used twice.